eva/configuration/configuration_manager.py

```diff
@@ -34,11 +36,25 @@ def __new__(cls):
 
         return cls._instance
 
+    @classmethod
+    def suffix_pytest_xdist_worker_id_to_dir(cls, path: Path):
+        try:
+            worker_id = os.environ["PYTEST_XDIST_WORKER"]
+            path = path / str(worker_id)
+        except KeyError:
+            pass
+        return path
+
     @classmethod
     def _create_if_not_exists(cls):
         if not cls._yml_path.exists():
+            initial_eva_config_dir = Path(EVA_DEFAULT_DIR)
```

Comment:
Please document the need for the logic. Thanks!

Reply:
Earlier, we directly used `EVA_DEFAULT_DIR`. Now, we prefix the worker id if needed. I will add a comment.